src/server/ApiRoutes/Feedback.js

```diff
@@ -1,25 +1,55 @@
-import axios from 'axios';
+import aws from 'aws-sdk';
 
 import appConfig from '../../app/data/appConfig';
+import { encodeHTML } from '../../app/utils/utils';
 
 export default {
   post: (req, res) => {
     if (!req.body) return res.status(400).json({ error: 'Malformed request' });
     if (!req.body.fields) return res.status(400).json({ error: 'Request body missing `field` key' });
 
+    aws.config.update({ region: 'us-east-1' });
+
     const { fields } = req.body;
-    return axios({
-      method: 'POST',
-      url: appConfig.feedbackFormUrl,
-      data: {
-        fields,
+
+    const submissionText = Object.entries(fields).map(([field, value]) => `${field}: ${encodeHTML(value)}`).join(', ');
+    const emailText = `Question/Feedback from Research Catalog (SCC): ${submissionText}`;
+
+    const emailHtml = `<div>
+    <h1>Question/Feedback from Research Catalog (SCC):</h1>
+    ${Object.entries(fields).map(([field, value]) => `<p>${field}: ${encodeHTML(value).replace(/\\n/g, '<br/>')}</p>`).join('')}
```

Comment:
I wonder if just "styling" this with a DL tag would look alright in an email client.. Might be a cheap way to more clearly delineate key/value

Reply:
Sorry to slow play these comments but xss prevention is tricky and things are occurring to me. I think it may be possible for a someone to craft their own post to this new endpoint with their own "fields" object, which could include any keys they like. We only want them to post "fields" with the following keys: "email", "feedback", and "url", right? So we should really assert that one is not able to add other fields. (i.e. theoretically they could add keys where the key itself (not the value) includes malicious html, which we'd just insert directly into the html part of the email.) If I'm reading this right, I'd just add a check to confirm the only keys allowed are those we expect.

Reply:
No worries. Thank you so so much for chiming in on this aspect of the story!
Would `['email', 'feedback', 'url'].map` in place of `Object.entries(fields)` fix this issue?

Reply:
Yes. Something like that sounds right

Reply:
Made that change in commit [0b92dbe](https://github.com/NYPL/discovery-front-end/pull/1587/commits/0b92dbe4956199b759530b813262b281e95d483b)